packages/lucide-static/scripts/generateSprite.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { stringify } from 'svgson';
+import { format } from 'prettier';
+import { appendFile } from '../../../scripts/helpers';
+
+export default function generateSprite(svgs, packageDir) {
+  const symbols = svgs.map(({ name, parsedSvg }) => ({
+    name: 'symbol',
+    type: 'element',
+    attributes: {
+      id: name,
+      viewBox: '0 0 24 24',
+      fill: 'none',
+      stroke: 'currentColor',
+      'stroke-width': '2',
+      'stroke-linecap': 'round',
+      'stroke-linejoin': 'round',
```

Comment:
Hi, Based on Discord conversation - having these on the symbol in the sprite prevents them from being properly set via CSS.
Not having them would mean that the end user would need to add them to their SVG import - either via CSS class or directly on the element.